airtime_mvc/composer.json

```diff
@@ -16,7 +16,7 @@
     "massivescale/celery-php": "^2.1",
     "php-amqplib/php-amqplib": "^2.12",
     "propel/propel1": "1.7.0-stable",
-    "raven/raven": "0.12.0",
+    "raven/raven": "^0.12",
```

Comment:
@paddatrapper @hairmare This is used to send logs to sentry, I don't know if it still used.
Based on the lack of documentation around it and that it seem to be a Airtime Saas tool, I assume it is not used, so I would suggest we remove it ?

Reply:
Not used in practice. May require some code changes to remove. Not sure if it is actually called anywhere